website/content/docs/agent-and-proxy/agent/index.mdx

```diff
@@ -176,22 +176,22 @@ These are the currently-available general configuration options:
 There can at most be one top level `vault` block, and it has the following
 configuration entries:
 
-- `address` `(string: <optional>)` - The address of the Vault server to
+- `address` `(string: <optional>)` - The address of the OpenBao server to
   connect to. This should be a Fully Qualified Domain Name (FQDN) or IP
-  such as `https://vault-fqdn:8200` or `https://172.16.9.8:8200`.
+  such as `https://openbao-fqdn:8200` or `https://172.16.9.8:8200`.
```

Comment:
Let's confirm that hostname.

Reply:
Hostname isn't defined by Bao, it is just used for a placeholder.